tests/test-cases/Group6-VIC-Machine/6-03-Delete.robot

```diff
@@ -138,6 +166,9 @@ Delete VCH with non-cVM in same RP
     Log  ${output}
     Should Be Equal As Integers  ${rc}  0
 
+    ${rc}  ${output}=  Run And Return Rc And Output  govc pool.destroy "%{TEST_RESOURCE}/Resources/%{VCH-NAME}"
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
```

Comment:
will cause leakage, change to
Run Keyword and Continue on Failure

Reply:
Sounds good, fixed.

Reply:
https://github.com/vmware/vic/pull/7686